lib/app/controllers/auth/auth_controller.dart

```diff
@@ -1,15 +1,23 @@
 import 'package:get/get.dart';
+import 'package:liga_inggris_mobile/services/auth/service.dart';
 
 class AuthController extends GetxController {
   AuthController();
-  var isLoggedIn = true.obs;
+  var isLoggedIn = false.obs;
+  final AuthService _authService = AuthService();
 
   // TODO: pengecekan email & password
-  void login() {
+  void login(String email, String password) async {
     isLoggedIn.value = true;
+    if (isLoggedIn.value) {
+      Get.toNamed('/');
+    }
+    await _authService.login(email, password);
+    isLoggedIn.value = _authService.isLoggedIn.value;
```

Comment:
```suggestion
    await _authService.login(email, password);
    isLoggedIn.value = _authService.isLoggedIn.value;
    Get.toNamed('/');
```
pindah ke bawah,
karena `isLoggedIn.value = true;`, berarti always true, tidak perlu if check